src/grid/Grid.js

```diff
@@ -15,35 +13,38 @@ class Grid extends Component {
     activeOpacity: 1,
   }
 
+  styles = StyleSheet.create({
```

Comment:
Ah I didn't see that. Ok that's fine. 

I can think of a couple other ways, but for this it seems like an overkill since only one stlye prop is being changed.